pkg/plugins/builtin/http/http.go

```diff
@@ -193,10 +195,23 @@ func exec(stepName string, config interface{}, ctx interface{}) (interface{}, in
 	if cfg.FollowRedirect != "" {
 		fr, err = strconv.ParseBool(cfg.FollowRedirect)
 		if err != nil {
-			return nil, nil, fmt.Errorf("failed to parse allow redirect: %s", err)
+			return nil, nil, fmt.Errorf("failed to parse follow_redirect: %s", err)
 		}
 	}
-	httpClient := httputil.NewHTTPClient(httputil.HTTPClientConfig{Timeout: td, FollowRedirect: fr})
+	var insecureSkipVerify bool
+	if cfg.InsecureSkipVerify != "" {
+		insecureSkipVerify, err = strconv.ParseBool(cfg.InsecureSkipVerify)
+		if err != nil {
+			return nil, nil, fmt.Errorf("failed to parse insecure_skip_verify: %s", err)
+		}
+	}
+	httpClient := httputil.NewHTTPClient(httputil.HTTPClientConfig{
+		Timeout:        td,
+		FollowRedirect: fr,
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: insecureSkipVerify},
```

Comment:
Not sure about what is a good DefaultTLSClientConfig to keep HTTP2 enabled here.